requirements-test.txt

```diff
@@ -1,12 +1,2 @@
-#
-# This file is autogenerated by pip-compile
-# To update, run:
-#
-#    pip-compile --output-file requirements-test.txt etc/requirements-test.in
-#
-
-click==7.0                # via pip-tools
```

Comment:
`click`, `pip-tools,` and `six` were not actually being used in any of the tests, so I removed them here.